Help_sources/Bosch/SerialHandler.py

```diff
@@ -0,0 +1,190 @@
+import serial,sys, time
+import threading
+
+
+class MessageConverter:
+    def MCTL(f_vel=0.0,f_angle=0.0):
+        if type(f_vel==float) and type(f_angle==float):
```

Comment:
line 7: type of an '==' evaluation is a BOOL, therefore the content of IF ... is always FALSE, therefore always empty is returned ...
Or no?

Reply:
The type of f_vel and f_angle will be float types in case of the default paramater, but we can pass strings if we want to, because python won't restrict us, so it won't be always empty. In case of giving the right parameters of float type it will return the expected value.